cli/internal/cache/cache_fs.go

```diff
@@ -25,17 +28,34 @@ func (f *fsCache) Fetch(target, hash string, _unusedOutputGlobs []string) (bool,
 
 	// If it's not in the cache bail now
 	if !fs.PathExists(cachedFolder) {
+		f.logFetch(false, hash)
 		return false, nil, nil
 	}
 
 	// Otherwise, copy it into position
 	err := fs.RecursiveCopyOrLinkFile(cachedFolder, target, fs.DirPermissions, true, true)
 	if err != nil {
+		// TODO: what event to log here?
 		return false, nil, fmt.Errorf("error moving artifact from cache into %v: %w", target, err)
 	}
+	f.logFetch(true, hash)
 	return true, nil, nil
 }
 
+func (f *fsCache) logFetch(hit bool, hash string) {
+	var event string
+	if hit {
+		event = "hit-local"
```

Comment:
@gaspar09 will it be easier in clickhouse queries of total time saved if we split this into `event: 'HIT'` and `source: 'LOCAL'`?

Reply:
Yea, more columns would be better. Otherwise we'd just need a  `startsWith` parsing or similar in an inner query.

Reply:
FWIW, We can also do this parsing on our backend.

Reply:
No problem, done.